doc/api/async_hooks.md

```diff
@@ -712,6 +712,75 @@ never be called.
 * Returns: {number} The same `triggerAsyncId` that is passed to the
 `AsyncResource` constructor.
 
+## Class: AsyncContext
+
+This class is used to create contexts that can be used through the event loop.
```

Comment:
The phrasing through the event loop isn't clear and I wouldn't have understood what it does from just reading that.

Maybe:

```suggestion
This class is used to create asynchronous state between callbacks and promise chain. It allows you to keep variables throughout the lifetime of a web request or any other asynchronous duration. It is similar to what ThreadLocal storage is in other languages.
```

Reply:
LGTM, thanks for the suggestion!